examples/src/main/java/io/opencensus/examples/stats/SignalFxStatsExample.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2017, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.examples.stats;
+
+import io.opencensus.common.Scope;
+import io.opencensus.exporter.stats.signalfx.SignalFxStatsExporter;
+import io.opencensus.stats.Aggregation;
+import io.opencensus.stats.Measure.MeasureDouble;
+import io.opencensus.stats.Measure.MeasureLong;
+import io.opencensus.stats.Stats;
+import io.opencensus.stats.View;
+import io.opencensus.stats.View.AggregationWindow;
+import io.opencensus.tags.TagContext;
+import io.opencensus.tags.TagKey;
+import io.opencensus.tags.TagValue;
+import io.opencensus.tags.Tagger;
+import io.opencensus.tags.Tags;
+import java.lang.management.ManagementFactory;
+import java.lang.management.MemoryMXBean;
+import java.lang.management.MemoryUsage;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.Collections;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Example application that reports its own heap usage as stats to SignalFx.
+ *
+ * <p>Measures every second the JVM's heap usage, maximum heap and an iteration count and reports
+ * those as 'jvm.heap.used', 'jvm.heap.max' and 'iterations' metrics. In this simple example, we use
+ * the measure name as the metric name because we only have one view on each of those measures.
+ *
+ * <p>This program runs forever until ^C-ed.
+ */
+public class SignalFxStatsExample {
```

Comment:
Can we move this into a different commit (and all the changes in examples)?

Reply:
Yup, I think I don't have a choice anyway because it's not going to build otherwise. I'll send a different PR later with the example once this is merged. Note that it doesn't seem possible to implement anything in `opencensus-examples` that depends on an unreleased version of `opencensus-java`. It might also make sense to break out the Java examples into their own repo to avoid the confusion between the builds.